Production/src/Selectors.cc

```diff
@@ -177,5 +179,123 @@ TauJetSelector::Result TauJetTag::Select(const edm::Event& event, const std::deq
     return Result(selected, nullptr);
 }
 
+TauJetSelector::Result TagAndProbe::Select(const edm::Event& event, const std::deque<TauJet>& tauJets,
+                                     const std::vector<pat::Electron>& electrons,
+                                     const std::vector<pat::Muon>& muons, const pat::MET& met,
+                                     const reco::Vertex& primaryVertex,
+                                     const pat::TriggerObjectStandAloneCollection& triggerObjects,
+                                     const edm::TriggerResults& triggerResults, float rho)
+{
+    const pat::Muon *ref_muon = nullptr;
+    const pat::Electron *ref_electron = nullptr;
+    for(const pat::Muon& muon : muons) { // Look for a muon tag first
+        if(!(muon.pt() > 26 && std::abs(muon.eta()) < 2.1 && muon.isMediumMuon() && PFRelIsolation(muon) < 0.1
```

Comment:
- pt > 28 GeV (to be the same for all eras)
- pf_rel_iso < 0.15 (which corresponds to the definition of the tight WP)